build/ci.go

```diff
@@ -275,6 +275,8 @@ func doTest(cmdline []string) {
 	// Get test fixtures.
 	csdb := build.MustLoadChecksums("build/checksums.txt")
 	downloadSpecTestFixtures(csdb, *cachedir)
+	downloadPragueSpecTestFixtures(csdb, *cachedir)
+	return
```

Comment:
Is `return` intended?

Reply:
Oh, this is a debug code. I must remove it. Thank you.